content/en/blog/_posts/2024-09-25-swap-ga-graduation/2024-09-25-swap-ga-graduation.md

```diff
@@ -247,13 +260,21 @@ the total physical swap capacity of the machine.
 is a Kubernetes addon for detecting hardware features and configuration.
 It can be utilized to discover which nodes are provisioned with swap.
 
-As an example, to figure out if a certain node is provisioned with swap,
-the following command can be executed while replacing `<NODE-NAME>` with the node's name:
+As an example, to figure out which nodes are provisioned with swap,
+use the following command:
 ```shell
-> k get node <NODE-NAME> -o yaml | grep "feature.node.kubernetes.io/memory-swap"
-      feature.node.kubernetes.io/memory-swap: "true"
+> kubectl get nodes -o jsonpath='{range .items[*]}{.metadata.name}: {.metadata.labels.feature\.node\.kubernetes\.io/memory-swap}{"\n"}{end}'
```

Comment:
```suggestion
kubectl get nodes -o jsonpath='{range .items[*]}{.metadata.name}: {.metadata.labels.feature\.node\.kubernetes\.io/memory-swap}{"\n"}{end}'
```

Reply:
Extra credit: handle missing annotations.

Reply:
Done + the extra credit :wink: